__tests__/nc-games.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require("supertest");
+const app = require ("../app.js");
+const seed = require("../db/seeds/seed.js")
+const data = require("../db/data/test-data/index")
+const db = require("../db/connection")
+
+beforeEach(() => {
+    return seed(data);
+})
+
+afterAll(() => {
+    db.end();
+})
+
+describe("/api/categories", () => {
+    describe("/GET", () => {
+
+        test("Status 200 returned when the GET HTTP method is used on endpoint", () => {
+            return request(app)
+            .get("/api/categories")
+            .expect(200)
+        })
+
+        test("Status 200: Returns all categories when the GET HTTP method is used on the categories endpoint", () => {
+            return request(app)
+            .get("/api/categories")
```

Comment:
You _could_ just pop your expect 200 assertion in with this bottom test. E.G: 
```
.get('/api/categories)
.expect(200)
.then(({body}) => {...etc})
```

And it would have much the same functionality.